modules/installation-machines-bare-metal.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-machines-bare-metal-{context}"]
+= Creating the cluster
+
+To create the {product-title} cluster, you provision machines by using the Ingition config
+files that you generated with the installation program.
+
+.Prerequisites
+
+* Your machines must have internet access.
+* Create the required infrastructure for the cluster.
+* You obtained the installation program and generated the Ignition config files
+for your cluster.
+
+.Procedure
+
+. Obtain the {op-system-first} images.
+
+. Use the the ignition config files that you generated by using the installation
+program to create the required bootstrap, worker, and master nodes.
+
+. Monitor for bootstrap-complete
+to monitor cluster bootstrapping.
++
+----
+$ openshift-install --dir test-bare-metal upi bootstrap-complete
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.12.4+c53f462 up
+INFO Waiting up to 30m0s for the bootstrap-complete event...
+----
++
+The command succeeds when the Kubernetes APIServer signals that it has been
+bootstrapped on the control plane machines.
+
+. Optionally, reformat the bootstrap machine.
```

Comment:
This should mention that the bootstrap node should be removed from the LB created above. This step is not optional.  (reformatting or reprovisioning the bootstrap machine is optional)